scanapi/reporter.py

```diff
@@ -43,6 +43,24 @@ def write(self, results):
         logger.info("\nThe documentation was generated successfully.")
         logger.info(f"It is available at {abspath(self.output_path)}")
 
+    def write_without_generating_report(self, results):
```

Comment:
```suggestion
    @staticmethod
    def write_without_generating_report(results):
```

Since `self` is not being used, I believe we can use the `staticmethod` decorator here. This would fix DeepSource.

Reply:
Thanks! :)